docs/structure.md

```diff
@@ -113,17 +134,28 @@ function.
         - `.place_order()`
 
     === "Storage"
+
+        The Storage class enables access to the UP42 storage. You can list
+        your assets and orders in storage.
+
         - `.get_orders()`
         - `.get_assets()`
     
     === "Order"
+
+        The Order class enables you to place, inspect and get information on orders.
+
         - `.info`
         - `.status`
         - `.metadata`
         - `.get_assets()`
         - `.track_status()`
     
     === "Asset"
+
+        The Asset class enables access to the UP42 assets in the user storage. Assets are results 
+        of orders or results of jobs with download blocks.
```

Comment:
I understand adding this info also here but I'm a bit worried about duplication of information in multiple locations and how this makes it complicated to maintain the docs. Not critical but good to keep in mind.

Reply:
Jup, working on nice solution via mkdocs-macros to have the description only once and to automate most of the "available functions" in the structure chapter + code reference. Not critical but wanted to check this library out in more detail anyway, it is so powerful. Will ask for full review when this is done.